geti_sdk/deployment/deployed_model.py

```diff
@@ -326,7 +350,18 @@ def load_inference_model(
                 raise ValueError(
                     f"Tiling is not supported for domain {self._converter.domain}"
                 )
-            self._tiler = tiler_type(model=model, execution_mode="sync")
+            classifier_name = "tile_classifier"
+            tiler_arguments = {"model": model, "execution_mode": "sync"}
+            if classifier_name in os.listdir(self.model_data_path):
+                classifier_path = os.path.join(self.model_data_path, classifier_name)
+                tile_classifier_model = model_api_Model.create_model(
+                    model=classifier_path + ".xml",
+                    weights_path=classifier_path + ".bin",
+                    core=None,
```

Comment:
```suggestion
                    core=core,
```
Should we reuse the existing core? If we loaded some plug-ins to this core we better use it further, it is not obvious though.

Reply:
Yeah good point, better to reuse it. I think it's safe to assume that any plugins that are loaded for the main model also apply to the tile classifier